docs/console/guides/blueprint/blueprint-configuration.md

```diff
@@ -221,6 +259,106 @@ Resource successfully created in project 'ash-devnet'!
+Resource successfully created in project 'ash-devnet'!
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------+
+| Resource name | Resource ID | Type            | Cloud region  | Size  | Created at       | Status  | Resource specific |
++===============+=============+=================+===============+=======+==================+=========+===================+
+| ash-subnet    | 396d...6cc8 | AvalancheSubnet | aws/us-east-1 | Small | 2024-01-30T17:26 | Pending |  ID         |     |
+|               |             |                 |               |       |                  |         |  Validators | 0   |
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------+
+```
+
+## Subnet ID
+
+After a few minutes, we can get the Subnet ID of our Subnet with the `console resource info` command:
+
+```bash title="Command"
+ash console resource info ash-subnet
+```
+
+```bash title="Output"
+Resource 'ash-subnet' of project 'ash-devnet':
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------------------------------------------------------+
+| Resource name | Resource ID | Type            | Cloud region  | Size  | Created at       | Status  | Resource specific                                                 |
++===============+=============+=================+===============+=======+==================+=========+===================================================================+
+| ash-subnet    | 396d...6cc8 | AvalancheSubnet | aws/us-east-1 | Small | 2024-01-30T17:26 | Running |  ID         | 2rArnpKkN9fKch9Q6ana3awAfVmM4HfLoNYrZZDkSumApEi8yZ  |
+|               |             |                 |               |       |                  |         |  Validators | 5                                                   |
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------------------------------------------------------+
+```
+
+## Blockchain ID
+
+From here, we can query one of our nodes to get information about the Subnet:
+
+```bash title="Command"
+ash avax subnet info 2rArnpKkN9fKch9Q6ana3awAfVmM4HfLoNYrZZDkSumApEi8yZ
+```
+
+```bash title="Output"
+Subnet '2rArnpKkN9fKch9Q6ana3awAfVmM4HfLoNYrZZDkSumApEi8yZ':
+  Type: Permissioned
+  Control keys: ["P-local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u"]
+  Threshold:    1
+  Blockchains list (1): 
+  - 'AshSubnetEVM':
+    ID:      2SZGABMprnB9Ux88WLYGodeWRFHJBogDrkncLDqsv5XhFnizT6
+    VM ID:   srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+    VM type: SubnetEVM
+  Validators list (5): 
+  - NodeID-MFrZFVCXPv5iCn6M9K6XduxGTYp891xXZ
+  - NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5
+  - NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg
+  - NodeID-GWPcbFJZFfZreETSoWjPimr846mXEKCtu
+  - NodeID-NFBbbJ4qCmNaCzeW7sxErhvWqvEQMnYcN
+```
```

Comment:
## Blockchain RPC endpoint

The CLI provides a powerful helper to get the RPC endpoint that can be used to query the Subnet EVM blockchain (e.g. to connect a Web3 wallet). Provide the `avalancheNode` resource to be used as RPC and the `avalancheSubnet`:

```bash title="Command"
ash console helper rpc local-node-01 my-subnet-evm
```

```bash title="Output"
AshLocalEVM RCP endpoint:
  http://44.223.28.33:9650/ext/bc/RfX8YRUHePxFENBZFwaNt1tP6CR6RQ4hJJ5AqYv49LRoSKniK/rpc
```

Reply:
Cannot do a suggestion on a README with code bocks

Reply:
Done in https://github.com/AshAvalanche/ash-docs/pull/79/commits/6df71bc71fe5f55f263dfd532ef4b6b885d72d2b